google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkService.java

```diff
@@ -27,7 +35,44 @@ public static CloudSdkService getInstance() {
     return ServiceManager.getService(CloudSdkService.class);
   }
 
+  @Nullable
   public abstract String getCloudSdkHomePath();
 
   public abstract void setCloudSdkHomePath(String path);
+
```

Comment:
By passing through to the common lib do you mean having the common lib expose this functionality for us? That is not a bad idea as I could see this being useful to other clients like ecplise.